src/lib/query/organizations/UsersByOrganization.js

```diff
@@ -0,0 +1,49 @@
+import gql from 'graphql-tag';
+
+export default gql`
+  # query usersByOrganization($id: Int!) {
+  #   users: usersByOrganization (organization: $id){
+  #     id
+  #     firstName
+  #     lastName
+  #     email
+  #     comment
+  #     groupRoles {
+  #       organization
+  #     }
+  #   }
+  # }
```

Comment:
This query `usersByOrganization` is the one that will list the users within an organization that are assigned to groups, this is what should be used to display users assigned to groups within an organization